scripts/g.extension/g.extension.py

```diff
@@ -675,8 +679,43 @@ def install_extension(source, url, xmlurl):
         if sys.platform == "win32":
             ret += install_extension_win(module)
         else:
+            # set /usr/bin/python to python3 cause of python shebangs
+            ps = gscript.Popen((
+                "update-alternatives", "--install", "/usr/bin/python",
```

Comment:
Do we need this part or would the change python -> python3 be enough?

Reply:
I don't think so. Please also note that `update-alternatives` will not work on Windows or possibly on some Linux distos.

Reply:
I removed the update-alternatives and replaces the shebangs for **not Windows** systems